lib/contentstack/version.rb

```diff
@@ -1,3 +1,3 @@
 module Contentstack
-  VERSION = "0.6.3"
+  VERSION = "0.6.4"
```

Comment:
Check versioning again. If we are adding new function then it should go for v0.7.0